runtime/v1/shim/client/client.go

```diff
@@ -157,7 +157,7 @@ func newSocket(address string) (*net.UnixListener, error) {
 }
 
 func connect(address string, d func(string, time.Duration) (net.Conn, error)) (net.Conn, error) {
-	return d(address, 100*time.Second)
+	return d(address, 10*time.Second)
```

Comment:
maybe we should add a const like `DefaultTimeout`, WDYT?

Reply:
yes, but I'm not sure if this is the root cause, so need some discuss.